public/app/features/alerting/unified/components/receivers/form/ChannelSubForm.tsx

```diff
@@ -96,7 +96,7 @@ export function ChannelSubForm<R extends ChannelValues>({
     <div className={styles.wrapper} data-testid="item-container">
       <div className={styles.topRow}>
         <div>
-          <Field label="Contact point type" htmlFor={contactPointTypeInputId} data-testid={`${pathPrefix}type`}>
+          <Field label="Type" htmlFor={contactPointTypeInputId} data-testid={`${pathPrefix}type`}>
```

Comment:
Perhaps `Service` or `Provider` would make more sense here from a user's perspective, as `type` is a somewhat technical abstraction?

Reply:
Integration?

Reply:
Integration?

<img width="1232" alt="Screenshot 2022-11-30 at 16 15 27" src="https://user-images.githubusercontent.com/85952834/204850712-1dab3fd4-4036-4283-ad0a-5da2fea7b028.png">

Reply:
That sounds good to me! 👌

Reply:
Yes!!! I agree. 
One receiver (contact point) can have 1 or more integrations. So, in this case, I think we should name it integration.

Reply:
I think that means then that `Integration` should be renamed back to `Type`, perhaps?

Reply:
That field is the `Type` of integration, rather than the integration as a complete piece of configuration?